.gitignore

```diff
@@ -24,3 +24,6 @@ __pycache__
 # Exception for images in assets folder
 !assets/*
 build/
+
+# Testing
```

Comment:
Given we don't have anything in the repo this change is likely unrequired.

If you need to ignore files locally you can do various things such as:
a) Simply don't git add them
b) Add them to your [user-local](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file) or [user-global](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) gitignore files

Reply:
i use vsc, theres no option of git-adding it. will keep this open until i've done testing.

Reply:
Then do the second option. I would recommend the user-local gitignore file

Reply:
VSC lets you stage changes, add files to a commit etc. Also the documentation I linked isn't related to just VSC its merely git. But yea, that works

Reply:
i tried and i cant